pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -395,19 +400,43 @@ func (m *managerImpl) processShutdownEvent() error {
 			}(pod, group)
 		}
 
+		var groupTerminationWaitDuration = time.Duration(group.ShutdownGracePeriodSeconds) * time.Second
```

Comment:
can you please add a note here in the code comment to explain why 1 second is added

Reply:
(this is retroactive improvement of code)

Reply:
Added comment.

Reply:
I was mostly asking why we are adding 1 second to the configured duration. Can you please add this comment

Reply:
sorry, going blind here.